core/src/components/segment/test/wrap/segment.e2e.ts

```diff
@@ -0,0 +1,149 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs().forEach(({ title, screenshot, config }) => {
+  test.describe(title('segment: wrap'), () => {
+    test('should not wrap short text', async ({ page }) => {
```

Comment:
I don't think the short text tests (both this and the toolbar one) are needed; the other segment tests already cover that.